### PR TITLE
Issue 9 10

### PR DESCRIPTION
**Pull request by rouazana**
Add a simple test to show how to test with several users

### CODE REVIEW

File: src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  val feeder = (0 until UserCount).map(i => Map("username" -> s"user$i", "password" -> "password")).toArray
+
+  val scn = scenario("Imap").feed(feeder)
+    .pause(1 second)
+    .exec(imap("Connect").connect()).exitHereIfFailed
+    .exec(imap("login").login("${username}", "${password}").check(ok))
+    .during(1 minute) {
+        exec(imap("list").list("", "*").check(ok, hasFolder("INBOX")))
+        .pause(200 milli)
+        .exec(imap("select").select("INBOX").check(ok))
```
Comment (chibenwa) [inline src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala]: No unselect of INBOX folder in the loop ?

File: src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        .pause(200 milli)
+        .exec(imap("fetch").fetch(Seq(Last()), AttributeList("BODY[HEADER]", "UID", "BODY[TEXT]")).check(no))
```
Comment (mbaechler) [inline src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala]: fetch should use a vararg

Comment (mbaechler) [inline src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala]: and I'd rather say Range.Last() or range = Last()

Comment (mbaechler) [inline src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala]: what means `check(no)`?

Comment (rouazana) [inline src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala]: > fetch should use a vararg

Not sure you can, as we are using an Expression[Seq[FetchRange]]

> and I'd rather say Range.Last() or range = Last()

of course you can, but for me Last() is crystal clear for a fetch command.

File: src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala (hunk 1 above)
Comment (rouazana) [inline src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala]: seems to be not needed: select is doing a close then re select

File: src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala (hunk 2 above)
Comment (rouazana) [inline src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala]: check that the answer is "NO"?

Comment (mbaechler) [inline src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala]: IMAP wording I guess

Comment (mbaechler) [inline src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala]: ok

Comment (rouazana) [inline src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapSimpleScenario.scala]: yes, there are essentially 3 responses: OK, NO, BAD (other are PREAUTH and BYE)

**Pull request merged**